smart_importer/predictor.py

```diff
@@ -40,7 +40,7 @@ class EntryPredictor(ImporterHook):
     weights: Dict[str, float] = {}
     attribute: Optional[str] = None
 
-    def __init__(self, predict=True, overwrite=False):
+    def __init__(self, predict=True, overwrite=False, string_tokenizer=None):
```

Comment:
This new argument document should be documented in the docstring above.

Reply:
@yagebu  thanks, done for this.